virtual-organizations/IceCube.yaml

```diff
@@ -49,3 +49,14 @@ PrimaryURL: https://grid-voms.desy.de:8443/voms/icecube/
 PurposeURL: http://www.icecube.wisc.edu/
 ReportingGroups:
 - IceCube
+
+DataFederations:
+  StashCache:
+    Namespaces:
+      /icecube/PUBLIC:
+        - ANY
+    AllowedCaches:
+      - ANY
```

Comment:
Did you run this by Benedikt?

Reply:
I guess I should ask if he wants to make the PUBLIC/PROTECTED split or not...

Reply:
Benedikt agrees with the split.